bom/application/pom.xml

```diff
@@ -3221,10 +3220,6 @@
                 <artifactId>jboss-logmanager-embedded</artifactId>
                 <version>${jboss-logmanager.version}</version>
                 <exclusions>
-                    <exclusion>
-                        <groupId>org.graalvm.nativeimage</groupId>
-                        <artifactId>svm</artifactId>
-                    </exclusion>
```

Comment:
Are you sure about that? Because AFAICS, it's still a dependency of `jboss-logmanager-embedded`.

Reply:
@dmlloyd ^^ fyi

Reply:
No, I'm not sure. I guess I'll have to keep `svm` in the bom for this exclusion to continue to work? Or would the preferred approach be to move to sdk for `jboss-logmanager-embedded` as well?

Reply:
This is an optional dependency and can be dropped from here; also, be aware that #33318 should minimize the awkward period wherein an exclusion would be needed.

Reply:
It should be fine so I'm OK with it (provided there are no testing issues of course).